android/app/src/main/res/layout/search_item_speaker.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
@iamareebjamal @shreyasnbhat Please reuse layouts. If you think the current layouts are bad, you can change them everywhere but try to reuse.